src/Benchmarks/Middleware/MiddlewareHelpers.cs

```diff
@@ -30,27 +29,15 @@ public static int GetMultipleQueriesQueryCount(HttpContext httpContext)
                     : 1;
         }
 
-        public static Task RenderFortunesHtml(IEnumerable<Fortune> model, HttpContext httpContext, HtmlEncoder htmlEncoder)
+        public static async Task RenderFortunesHtml(IEnumerable<Fortune> model, HttpContext httpContext,
+            HtmlEncoder htmlEncoder, SliceFactory<IEnumerable<Fortune>> fortunesFactory)
         {
             httpContext.Response.StatusCode = StatusCodes.Status200OK;
             httpContext.Response.ContentType = "text/html; charset=UTF-8";
 
-            var sb = StringBuilderCache.Acquire();
```

Comment:
Can we get rid of `StringBuilderCache` then? Haven't checked if anything else uses it.

Reply:
https://github.com/aspnet/Benchmarks/blob/05ad4582116b62443cefe5bef4086bfe159d02a2/src/Benchmarks/Data/BatchUpdateString.cs#L28